src/main/resources/views/CommercialVehicles/IVA30.hbs

```diff
@@ -1,131 +1,236 @@
+            <td></td>
+        </tr>
+    </table>
+
+    <!-- Required Standards Not Met Section -->
+    <div class="section-border">
+        <div class="section-title-border">
+            <h1> DEFECT DETAILS </h1>
         </div>
-        <!-- second box -->
-        <div class="section-border">
-            <div class="section-title-border">
-                <h1>USE OF THE VEHICLE</h1>
-            </div>
-            <div class="section-text-wrapper">
-                <p id="no-margin-p">It is an offence to use a vehicle that requires type approval without an Individual Approval
-                    Certificate confirming compliance with the type approval requirements, except for the purposes of:</p>
-                <ul class="vehicle-section-list">
-                    <li> - submitting it, by previous arrangement, for an approval examination;</li>
-                    <li> - bringing it away from such an examination;</li>
-                    <li> - delivering it, by previous arrangement, to a place where relevant work is to be done on it following
-                        receipt of a refusal to issue an Individual Approval Certificate;</li>
-                    <li> - bringing it away from a place where relevant work has been done on it.</li>
+        <div class="section-title-border" id="defect-details-subtitle-border">
+            <h1> Required Standard(s) Not Met: </h1>
+        </div>
+        <div class="section-text-wrapper">
+            <!-- List incoming defects here -->
+        </div>
+    </div>
+
+    <p> I hereby refuse an Individual Approval Certificate in respect of the above described vehicle on the
+        grounds that the vehicle fails to comply with the relevant requirements prescribed under The Road Vehicles
+        (Approval) Regulations 2020 </p>
+
+    <!-- Date/name table here -->
+    <table class="inspector-detail-table">
+        <tr>
+            <td></td>
+            <td class="table-row-cell"> <b>Reapplication Required By: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell"> <b>Signed: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"> <b>Date: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell"> <b>Name: </b> <!--Put data here --> </td>
+            <td class="table-row-cell"> <b>Station: </b> <!--Put data here --> </td>
+        </tr>
+        <tr>
+            <td class="table-row-cell" colspan="2"> <p><b>The Following additional defects may result in the vehicle not complying with the requirements
+                of the Road Vehicles (Construction and Use) Regulations 1986. (See notes overleaf) </b></p>
+                <!-- Put AdditionalDefects here -->
+            </td>
+        </tr>
+    </table>
+
+
+    <!-- Data Protection Section -->
+    <div class="section-border" id = "data-prot-section">
```

Comment:
Original cert has this entire box bolded, not just the 'header'. 

Header appears to have a separate more bold box that sits ever so slightly inside the main data protection statement box.